Xamarin.Forms.Platform.UAP/WebViewRenderer.cs

```diff
@@ -401,5 +417,17 @@ void UpdateCanGoBackForward()
 			((IWebViewController)Element).CanGoBack = Control.CanGoBack;
 			((IWebViewController)Element).CanGoForward = Control.CanGoForward;
 		}
+
+		void OnSeparateProcessLost(Windows.UI.Xaml.Controls.WebView sender, WebViewSeparateProcessLostEventArgs e)
```

Comment:
What are the conditions that cause this to get called?
I tried a few different things but it never seems to fire. 

Also does this need to unwire the previous webview when this is called? Is there a previous webview when this is called?

Reply:
I'm not certain what conditions would cause this to happen.  the [documentation](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.webview?view=winrt-19041#execution-modes) don't elaborate other than suggesting that it is a possibility. I'm _guessing_ if the process is killed or crashed this would happen.
The sample given in the documentation does not unwire the event handlers - i based the implementation off this.